doc/example-config

```diff
@@ -612,6 +612,18 @@
 #
 # mark-as-read-on-hover yes
 
+####  max-browser-tabs
+#
+# Sets the maximum number of article that can be opened by the open in the 
```

Comment:
"Article*s*", not "article". I prefer to copy and paste the descriptions from configcommands.dsv—it's simply easier.